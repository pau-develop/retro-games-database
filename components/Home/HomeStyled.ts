import styled from "styled-components";

const HomeStyled = styled.div`
  h1 {
    color: red;
  }
`;

export default HomeStyled;
