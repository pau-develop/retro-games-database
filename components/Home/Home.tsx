import HomeStyled from "./HomeStyled";

const Home = () => {
  return (
    <HomeStyled>
      <h1>Hello</h1>
    </HomeStyled>
  );
};

export default Home;
