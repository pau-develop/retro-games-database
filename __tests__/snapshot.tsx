import { render } from "@testing-library/react";
import Home from "../components/Home/Home";

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
