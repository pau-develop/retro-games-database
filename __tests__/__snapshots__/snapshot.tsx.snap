// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="sc-bcXHqe dOnZML"
  >
    <h1>
      Hello
    </h1>
  </div>
</div>
`;
